src/gui/dialogs/campaign_selection.cpp

```diff
@@ -120,7 +120,7 @@ void campaign_selection::campaign_selected()
 					// so we want to prepend each label to its description here
 					desc = cfg["label"].str() + "\n" + desc;
 
-					entry["details"] = std::move(desc);
+					entry["details"] = desc;
```

Comment:
I don't think this change is needed anymore. `config_attribute_value` now accepts `string&&`.